### PR TITLE
Update default_patterns to base

### PR DESCRIPTION
**Pull request by gmoro**
minimal_base now has too few packages

### CODE REVIEW

File: control/control.leanos.xml [hunk 1]
```diff
@@ -111,7 +111,7 @@ textdomain="control"
 
         <!-- bnc#875350: Explicitly selecting these patterns by default -->
         <!-- For a list of patterns, see https://build.suse.de/package/view_file/SUSE:SLE-12-SP3:GA/patterns-sles/patterns-sles.spec?expand=1 -->
-        <default_patterns>minimal_base</default_patterns>
+        <default_patterns>base</default_patterns>
```
Comment (jreidinger) [inline control/control.leanos.xml:114]: this is the set of patterns that are always selected by proposal. But any product can overwrite it in its skelcd, but I do not see yet any module doing it, so it use this shared one.
Change in general looks good for me. Just please add entry to changelog(including bzilla or jira entry ) and bump version.

**Comment by dgdavid** [comment]
@gmoro 

Hi! Just a friendly ping to check whether you got the @jreidinger review :)

**Comment by gmoro** [comment]
> @gmoro
> 
> Hi! Just a friendly ping to check whether you got the @jreidinger review :)

Hi,
The original changes to the patterns was postponed, will reactivate this change later when we get to merge the rest. Sorry to leave it open here and not tell you guys.

**Comment by dgdavid** [comment]
> Sorry to leave it open here and not tell you guys.

No problem. I'm going to mark it as draft to stop receiving it as part of the ["pending prs" weekly notification](https://github.com/jreidinger/pull_requests_reminder).

Ping us when it is ready again.

**Comment by shundhammer** [comment]
No action since 11/2021. Closing.

**Pull request closed without merging**